tests/test_ecs.py

```diff
@@ -288,6 +291,7 @@ async def test_network_mode_default(aws_credentials, launch_type: str):
         auto_deregister_task_definition=False,
         command=["prefect", "version"],
         launch_type=launch_type,
+        image="prefecthq/prefect:2.1.0-python3.8",
```

Comment:
We should not require this to be passed explicitly and it should not be included in all these tests.